client/app/collections/duplicates.coffee

```diff
@@ -7,11 +7,15 @@ module.exports = class Duplicates extends Backbone.Collection
 
     model: require 'views/models/merge'
 
-    url: null # Local collection.
+    # Local collection
+    url: null
 
     # Initialize the collection with duplicates fund in the sepcified contacts
     # collection.
-    findDuplicates: (collection) ->
+    initialize: ([], options) ->
```

Comment:
The `Duplicates` collection extends the `Backbone.Collection` class, which takes 2 args:
1. an array (empty by default) containing the defaults models to populate the collection instance with
2. an option hash

This class is specific because it takes an underlying collection (passed as `option.collection`), and parse it to find potential duplicates and populate itself with the duplicates found. So, to keep the collection in the option hash accessible, I take default args for the `initialize` method just like the original `Backbone.Collection` class :smiley:.

Note that I'm not especially fan of this method, and a revamp of the merge process will probably result of another way to deal with duplicates rather than this collection :wink:.